lib/perfdata/influxdbwriter.cpp

```diff
@@ -329,6 +338,9 @@ void InfluxdbWriter::CheckResultHandlerWQ(const Checkable::Ptr& checkable, const
 		fields->Set("acknowledgement", new InfluxdbInteger(checkable->GetAcknowledgement()));
 		fields->Set("latency", cr->CalculateLatency());
 		fields->Set("execution_time", cr->CalculateExecutionTime());
+		//fields->Set("short_message", CompatUtility::GetCheckResultOutput(cr));
+		//fields->Set("full_message", cr->GetOutput());
+		fields->Set("check_source", cr->GetCheckSource());
```

Comment:
That's somewhat redundant information - the above code adds the check source into each metric, and here it is added again. Also, the commented code lines shouldn't have been added to the PR, right?

Reply:
Basically we agree that this is redundant information, however, the check source (as part of the performance data) is only stored if there actually is some performance data returned by the check plugin. The check source in the metadata is always written (if enabled in the config), i.e., the check source is also available if the check plugin does not provide any performance data at all. It also may help debugging because one can figure out which check source did not provide performance data although it should.

Regarding the commented lines: we are not sure how the `short_message` and `full_message` should be handled in the general case. In accordance to the other perfdata writers (e.g., gelf) they probably should be included, however, these messages are not very useful in a timeseries database. For us it turned out to be a waste of disk space without any actual benefit, so for now we removed it again.